Gems/EMotionFX/Code/EMotionFX/Source/EventDataFloatArray.h

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+#include <EMotionFX/Source/EventData.h>
+
+
+namespace AZ
+{
+    class ReflectContext;
+}
+
+namespace EMotionFX
+{
+    class EMFX_API EventDataFloatArray
```

Comment:
suggestion: An API comment for this would definitely be helpful (also we might want to make a follow-up doc task to document this new event data type).